src/WingetCreateCLI/Commands/BaseCommand.cs

```diff
@@ -288,6 +288,12 @@ protected static async Task<string> DownloadPackageFile(string installerUrl)
                     return null;
                 }
 
+                if (e is InvalidDataException)
+                {
+                    Logger.ErrorLocalized(nameof(Resources.DownloadFileExceedsMaxSize_Error), e.Message);
```

Comment:
Instead of just putting this in the message i rather like making these strange typed values on the exception, like what you did for the other exception. In that manner, we can still jhave a meaningful message on the exception for debugging, but the data the caller needs to derive a good localized error message can still be handed out?

Reply:
I created a new exception for DownloadSizeExceeded similar to the other exceptions I created so that the data value isn't hidden in the exception message.

Reply:
Lol.. "Strong" not strange   :)